src/constants.js

```diff
@@ -193,25 +184,14 @@ export const SAUCE_CONNECT_CLI_PARAMS = [{
     name: 'max-logsize',
     description: 'Rotate logfile after reaching <bytes> size. Disabled by default.',
     type: 'number'
-}, {
-    alias: 'M',
-    name: 'max-missed-acks',
-    description: 'The maximum amount of keepalive acks that can be missed before the client will trigger a reconnect. (default 30)',
-    type: 'number',
-    default: 30
 }, {
     name: 'metrics-address',
-    description: 'host:port for the internal web server used to expose client side metrics. (default "localhost:8888")'
+    description: 'host:port for the internal web server used to expose client side metrics. (default "localhost:8888")',
+    deprecated: true
 }, {
     name: 'no-autodetect',
     description: 'Disable the autodetection of proxy settings.',
     type: 'boolean'
-}, {
-    alias: 'N',
-    name: 'no-proxy-caching',
-    description: 'Disable caching in Sauce Connect. All requests will be sent through the tunnel.',
-    type: 'boolean',
-    deprecated: true
```

Comment:
I am less concerned about removing deprecated options, though it will still require a major release on our part.  That's not a huge deal, but we should be clear about what we're doing

Reply:
Im removing flags that should have never been added. This one, for example, has been disabled for a long time now in SC and doesn't do anything.
I think it's worth releasing a major, instead of leaving the dysfunctional option.

Reply:
I mean, yeah, if it's literally broken anyway and nobody's complaining about it...¯\\\_(ツ)\_/¯